she_codes_news/templates/base.html

```diff
@@ -1,25 +1,32 @@
+{% load static %}
 <!doctype html>
 
 <html lang="en">
     <head>
         <meta charset="utf-8">
         <title>She Codes News</title>
         <meta name="description" content="">
-        <meta name="author" content="Hayley van Waas">
+        <meta name="author" content="Ellen">
         <link href="https://fonts.googleapis.com/css?family=Manjari&display=swap" rel="stylesheet">
-        <link rel="stylesheet" type="text/css" href="">
+        <link rel="stylesheet" type="text/css" href="{% static 'styles.css' %}">
+        {% block stylesheets %}{% endblock %}
     </head>
 
     <body>
         <header>
             <div id="header-text">
                 <a id="header-link" href="{% url 'news:index' %}">
-                    <h1>She {Codes} News</h1>
+                    <h1 style="color:#8246af">She<span style="color: #f9a427">{Codes} News</span></h1>
```

Comment:
nice custom colours, but i would have considered putting this in a class in the css file